docs/src/tutorials/optimizations.md

```diff
@@ -0,0 +1,119 @@
+# Stochastic optimization with discreteness
+
+```@setup random_walk
+import Pkg
+Pkg.activate("../../../tutorials/toy_optimizations")
+Pkg.develop(path="../../..")
+Pkg.instantiate()
+```
+
+In this tutorial, we solve two stochastic optimization problems using `StochasticAD` where the objective contains discrete randomness. We will need the following packages:
```

Comment:
"the objective contains" sounds a bit imprecise.

Reply:
"involves samples from discrete distributions"?